### PR TITLE
[#212] Update mux_selector_server.py and add rosRoverStartLocal.launch

### LINKED ISSUE #212: Create a roslaunch file meant to run the packages locally and launch the mux_selector in local mode
Create a launch file in `robotics/robot/util` named `rosRoverStartLocal.launch`.

Include the mux_selector and pass it the argument "local". You should check in `mux_selector_server.py` in the mux_selector rospackage to make sure you're sending it the right argument

### PR DESCRIPTION
**Pull request by AFknUsername**
closes #212 

Modify mux_selector_server.py to process correct number of args.
Add rosRoverStartLocal.launch which launches mux which arg "local".

### CODE REVIEW

File: robot/rospackages/src/mux_selector/scripts/mux_select_server.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def select_device(device):
 def mux_select_server():
     global local
     local = False
-    if len(sys.argv) == 2:
+    if len(sys.argv) > 3:
```
Comment (MartensCedric) [inline robot/rospackages/src/mux_selector/scripts/mux_select_server.py]: This scheme works well with the launch file. Although when you want to run it in standalone, if you try `rosrun mux_selector mux_selector_server.py local` then it no longer works. What I recommend is scraping this scheme of checking the amount of arguments and instead iterating over each argument to check if it's an exact match of "local"

**Comment by MartensCedric** [comment]
Reminder to me to fix commit a66f212 to ensure that github links it properly to @AFknUsername's account

File: robot/rospackages/src/mux_selector/scripts/mux_select_server.py [hunk 2]
```diff
@@ -115,14 +115,10 @@ def select_device(device):
 def mux_select_server():
     global local
     local = False
-    if len(sys.argv) == 2:
-        if sys.argv[1] == "local":
+    for x in sys.argv:
+        if x == "local":
             print("Running in local mode")
             local = True
-        else:
```
Comment (PeterGhimself) [inline robot/rospackages/src/mux_selector/scripts/mux_select_server.py]: I don't think this error message should be removed. Makes it harder to debug when the time will come again.

**Comment by MartensCedric** [comment]
Running
`roscore`
`rosrun mux_selector mux_select_server.py`
`rosrun mux_selector mux_select_server.py local`
`./mux_select_server.py`
`./mux_select_server.py`

Work

But there exists some need of whitelisting some arguments from the .launch file

**Comment by MartensCedric** [comment]
```
Error with passed arguments: 
__name:=mux_select_server
__log:=/home/cedric/.ros/log/26d5da4c-22c9-11ea-9b3c-3ca9f4b3afc0/mux_select_server-1.log
Exiting...
```

**Comment by MartensCedric** [comment]
I fixed the issue with ROS arguments

**Comment by MartensCedric** [comment]
@AFknUsername The PR is approved by me and also @PeterGhimself (IRL)

**Comment by AFknUsername** [comment]
#can't merge cause unresolved issues

**Comment by MartensCedric** [comment]
there's no conflicts but @PeterGhimself needs to approve I think too.

**Pull request merged**